.github/workflows/release-ext.yml

```
@@ -87,6 +87,13 @@ jobs:
           name: onekey-extension-rn
           path: |
             ./apps/ext/build/_dist/*.zip
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)